public/js/main.js

```diff
@@ -158,10 +158,22 @@ async function initialize() {
   return Promise.all([initCamera(), initializeModel()]);
 }
 
+function cleanupHtml() {
+  // eslint-disable-next-line no-restricted-globals
+  if (location.hostname !== 'localhost' && location.hostname !== '127.0.0.1') {
+    const reloadScript = document.querySelector(
+      'script[src="/reload/reload.js"]'
+    );
+    reloadScript.parentElement.removeChild(reloadScript);
+  }
+}
+
 /* Initializes necessary components and starts the
  * inference engine.
  */
 async function main() {
+  cleanupHtml();
```

Comment:
Is it ok if this is done in parallel to initializing the camera and models? if yes, can you rename cleanuphtml -> initializeDom (or similar) and add it to the list of promises in initialize()?

Reply:
Ooh yeah! Brilliant idea!
I'll do that once I'm happy with the E2E/Cypress stuff I've done so far.